requirements.txt

```diff
@@ -1 +1,8 @@
+# Minumum requirements
+#
+# Check these against
+#   nibabel/info.py
+#   .travis.yml
+#   doc/source/installation.rst
```

Comment:
Not for this PR but just saw these notes inspired by duplication, so decided to ask ;-)